src/main/java/luke/Luke.java

```diff
@@ -0,0 +1,78 @@
+package luke;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class Luke {
+
+    //Logo created using https://patorjk.com/software/taag/#p=display&f=Varsity&t=Luke
+    private static String logo = "  _____             __             \n"
+            + " |_   _|           [  |  _         \n"
+            + "   | |     __   _   | | / ] .---.  \n"
+            + "   | |   _[  | | |  | '' < / /__\\\\ \n"
+            + "  _| |__/ || \\_/ |, | |`\\ \\| \\__., \n"
+            + " |________|'.__.'_/[__|  \\_]'.__.' ";
+
+    public static void main(String[] args) throws LukeException {
+        Storage storage;
+        UI ui = new UI();
+        ui.greet();
+
+        String wd = System.getProperty("user.dir");
```

Comment:
The process of initialising the storage file may be more suitable to be done by the Storage class.